packages/modules/mssqlserver/src/mssqlserver-container.ts

```diff
@@ -0,0 +1,123 @@
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer } from "testcontainers";
+
+const MSSQL_PORT = 1433;
+
+type ProductKey = `${string}-${string}-${string}-${string}-${string}`;
+
+type Edition = "Evaluation" | "Developer" | "Express" | "Web" | "Standard" | "Enterprise" | ProductKey;
+
+const PASSWORD_CATEGORY_VALIDATION_PATTERNS = [/[A-Z]+/, /[a-z]+/, /[0-9]+/, /[@$!%*?&]+/];
+
+export class MSSQLServerContainer extends GenericContainer {
+  private database = "master";
+  private username = "SA";
```

Comment:
See https://github.com/testcontainers/testcontainers-java/pull/7238#issuecomment-1620777580

```suggestion
  private username = "sa";
```

Reply:
Thank you for your comments! This is actually my first time contributing so I really appreciate the feedback.